types/api.yaml

```diff
@@ -45,6 +45,20 @@ ValidatorBalanceResponse:
       $ref: "./primitive.yaml#/Gwei"
       description: "Current validator balance in gwei."
 
+ValidatorIdentityResponse:
+  type: object
+  required: [index, pubkey, activation_epoch]
+  properties:
+    index:
+      $ref: './primitive.yaml#/Uint64'
+      description: "Index of validator in validator registry."
+    pubkey:
+      $ref: './primitive.yaml#/Pubkey'
+      description: "Public key of validator."
+    activation_epoch:
```

Comment:
Why add the `activation_epoch`? It's a curious in-between of no epoch and all epochs in the validator record

Reply:
Index re-use won't come until years down the line if ever. Is it better to do a v2 in 2030 when that happens?

Reply:
i was thinking there's no harm having it?